modules/otel-config-instrumentation.adoc

```diff
@@ -280,6 +349,14 @@ The `+instrumentation.opentelemetry.io/otel-go-auto-target-exe+` annotation sets
 [id="otel-configuration-of-python-auto-instrumentation_{context}"]
 == Configuration of the Python auto-instrumentation
 
+:FeatureName: The Python auto-instrumentation
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+By default, this feature injects unsupported, upstream instrumentation libraries.
```

Comment:
Same comment

```suggestion
By default, this feature injects unsupported upstream instrumentation libraries.
```

Reply:
Thank you for the suggestion. Using a comma here is correct in my understanding of punctuation.